src/components/topic/partials/topic.html

```diff
@@ -2,6 +2,6 @@
   <div class="ga-topic-item" ng-class="{'ga-topic-active': topic.id == activeTopic}" ng-repeat="topic in
   topics" ng-click="setActiveTopic(topic.id)" data-original-title="{{topic.tooltip | translate}}">
     <div translate>{{topic.id}}</div>
-    <img ng-src="{{topic.thumbnail}}" alt="" height="60" width="110" draggable="false" >
+    <div class="sprite sprite-{{topic.id}}" alt="" height="60" width="110" draggable="false" ></div>
```

Comment:
Can you add a prefix `ga-topics-sprite`? , in case we will have multiple sprite on the page in the future